### PR TITLE
Add action hooks to commissions user meta table

### PR DESCRIPTION
**Pull request by davidsherlock**
As per my issue #302.

### CODE REVIEW

File: includes/user-meta.php [hunk 1]
```diff
@@ -20,6 +20,7 @@ function eddc_user_paypal_email( $user ) {
 	<h3><?php _e('Easy Digital Downloads Commissions', 'eddc'); ?></h3>
 	<table class="form-table">
 		<?php if ( current_user_can( 'manage_shop_settings' ) ) : ?>
+		<?php do_action( 'eddc_user_meta_table_start', $user ); ?>
```
Comment (cklosowski) [inline includes/user-meta.php]: Can we switch this to `eddc_user_profile_table_start`? That way it falls in line with the hooks for WP core which are `show_user_profile` and `edit_user_profile`. While, they do end up in user meta, the hooks themselves should be easy to identify where they are being used.

File: includes/user-meta.php [hunk 2]
```diff
@@ -51,6 +52,7 @@ function eddc_user_paypal_email( $user ) {
 				<span class="description"><?php _e( 'Check this box if you wish to prevent sale notifications from being sent to this user.', 'easy-digital-downloads' ); ?></span>
 			</td>
 		</tr>
+		<?php do_action( 'eddc_user_meta_table_end', $user ); ?>
```
Comment (cklosowski) [inline includes/user-meta.php]: Same as above, using `profile` instead of `meta`

**Comment by cklosowski** [comment]
I've fixed the issue in `master` that is causing the build to fail, it is unrelated to these changes.

**Comment by davidsherlock** [comment]
Done @cklosowski

**Comment by davidsherlock** [comment]
@cklosowski, can you merge this one please if you are good with it? Keen to improve my extension UI in the next commissions point release.

**Pull request merged**